package.json

```diff
@@ -28,8 +28,11 @@
     "@types/webpack-env": "^1.15.1",
     "axios": "^0.19.2",
     "clsx": "^1.1.0",
+    "moment": "^2.26.0",

```

Comment:
Probably don't need to specify it in here since `moment-timezone` requires moment anyway